src/rum.js

```diff
@@ -49,12 +49,13 @@ if (!rumConfig.serviceName) {
 if (!rumConfig.serverUrl) {
     rumConfig.serverUrl = serverUrl
 }
-if (!rumConfig.serviceVersion) {
-    rumConfig.serviceVersion = serviceVersion
-}
-rumConfig.logLevel = 'debug'
 
-var apm = initApm(rumConfig)
+const apm = initApm(Object.assign({}, rumConfig, {
+    serviceVersion,
+    logLevel: 'debug',
+    breakdownMetrics: true,
+    centralConfig: true,
```

Comment:
Central config will fail on older versions of the APM server.

Reply:
opbeans are always built from maser, is it a concern? I will enable breakdown metrics alone then.

Reply:
Even that would break if we test on old versions.

Reply:
All opbeans are tested aganist the latest stack versions. So this is not a blocker for us. Lets go ahead and enable both breakdown and centralConfig.